tests/ManagerTest.php

```diff
@@ -0,0 +1,97 @@
+<?php
+
+namespace PodPoint\Reviews\Tests;
+
+use PodPoint\Reviews\Exceptions\ProviderConfigNotFoundException;
+use PodPoint\Reviews\Exceptions\ProviderNotFoundException;
+use PodPoint\Reviews\Manager;
+
+class ManagerTest extends TestCase
```

Comment:
Class DocBlock

To be honest for test suite I see on MIS we don't add DocBlocks for test classes, so I think you can remove in the test above where you did add one.

Details :sweat_smile: :shrug: